### ISSUE TITLE
Icon request: icon-sign-up

### ISSUE THREAD

**Issue by mgutz** [FortAwesome/Font-Awesome#872]
It strikes me as odd that one of the most frequent things you do on a web site is sign up, yet I can't find a decent sign up icon. Any recommendations?

**Comment by davegandy** [comment]
I think sign up is a separable concept from user-add. Going to add this to 5.0 and close #2103. :)

**Comment by LordCemOnur** [comment]
This looks like an old issue but I also came across this when looking for an icon that is consistent/harmonious with default sign-in/sign-out icons. So I made the custom icon below to use in my project but feel free to include it in FA if it meets your design criteria.

![sign-up](https://user-images.githubusercontent.com/33794346/54948192-d43ca900-4f1a-11e9-8dd8-8adc0598b20e.png)
`sign-up`

![comparison](https://user-images.githubusercontent.com/33794346/54948785-48c41780-4f1c-11e9-80ba-b8350bc3d702.png)
`sign-up` icon with `sign-in` and `sign-out` icons for reference.
